files/bootstrap.sh

```diff
@@ -555,6 +555,7 @@ if [[ "$CONTAINER_RUNTIME" = "containerd" ]]; then
   fi
 
   sudo mkdir -p /etc/containerd
+  sudo mkdir -p /etc/containerd/config.d
```

Comment:
Do we need this if it's created at build time?

Reply:
bottom line, probably not, but its still a contract set with the bootstrap.sh and not the build so i figure its safe to have in both 🤔

Reply:
Fair enough!